mLRS/CommonRx/mavlink_interface_rx.h

```diff
@@ -448,83 +446,95 @@ if(txbuf>90) dbg.puts("-20 "); else dbg.puts("0   ");
 }
 #endif
 
-
+// This method should be selected for PX4 and currently may be a useful alternative for Ardupilot
 bool MavlinkBase::handle_txbuf_px4(uint32_t tnow_ms)
```

Comment:
could you pl rename this function to "xxx_method_b1"
unless you want to keep it "_px4" to emphasize it's default nature for px4, but when I would argue also the option should be named "px4" and not "mth_b1" :)